extensions/kafka/src/main/java/io/deephaven/kafka/ingest/JsonNodeObjectFieldCopier.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.kafka.ingest;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import io.deephaven.chunk.ObjectChunk;
+import io.deephaven.chunk.WritableChunk;
+import io.deephaven.chunk.WritableObjectChunk;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.deephaven.chunk.attributes.Values;
+
+public class JsonNodeObjectFieldCopier implements FieldCopier {
```

Comment:
Does it make sense to call this a `JsonNodeNodeFieldCopier` instead of "Object"?

Reply:
Done.